web/packages/extension/src/options.ts

```diff
@@ -3,5 +3,7 @@ import { bindOptions } from "./common";
 
 window.addEventListener("DOMContentLoaded", () => {
     document.title = utils.i18n.getMessage("settings_page");
+    document.querySelector("#advanced_options")!.textContent =
```

Comment:
nit: Personally I prefer the good old `document.getElementById()` over `document.querySelector("#some_id")`. But that's debatable of course.